backend/src/main/java/com/woowacourse/kkogkkog/application/CouponService.java

```diff
@@ -116,10 +119,10 @@ private Coupon findCoupon(Long couponId) {
             .orElseThrow(CouponNotFoundException::new);
     }
 
-    private void saveMemberHistory(Member hostMember, Member targetMember, CouponType couponType,
+    private void saveMemberHistory(Member hostMember, Member targetMember, Coupon coupon,
                                    CouponEvent couponEvent) {
-        MemberHistory memberHistory = new MemberHistory(null, hostMember, targetMember, couponType,
-            couponEvent);
+        MemberHistory memberHistory = new MemberHistory(null, hostMember, targetMember, coupon.getId(),
+            coupon.getCouponType(), couponEvent, coupon.getMeetingDate());
```

Comment:
별도의 생성자를 만드는 건 어떨까요?

```suggestion
        MemberHistory memberHistory = new MemberHistory(hostMember, targetMember, coupon, couponEvent);
```